docs/authenticate.js

```diff
@@ -145,32 +140,10 @@ class Authenticator {
   internalInitializeEE() {
     initializeEE(this.eeInitializeCallback, (err) => {
       if (err.message.includes('401') || err.message.includes('404')) {
-        // HTTP code 401 indicates "unauthorized".
-        // 404 shows up when not on Google internal network.
-        // TODO(#340): Maybe don't require EE failure every time, store
-        //  something in localStorage so that we know user needs token. Then
-        //  tests can just set that as well, unifying the codepaths.
-        const dialog = $(eeErrorDialog).dialog({
-          buttons: [
-            {
-              text: 'Sign in with EarthEngine-enabled account',
-              click: () => this.requireSignIn(),
-            },
-            {
-              text: 'Continue without sign-in',
-              click: () => {
-                // Don't trigger close callback, but close dialog.
-                dialog.dialog({close: () => {}});
-                dialog.dialog('close');
-                this.getAndSetEeTokenWithErrorHandling().then(
-                    () => initializeEE(
-                        this.eeInitializeCallback, defaultErrorCallback));
-              },
-            },
-          ],
-          modal: true,
-          width: 600,
-          close: () => this.requireSignIn(),
+        showToastMessage('fetching ee token', -1);
```

Comment:
Maybe expand this a bit? "Fetching EarthEngine token" at least? But maybe something like "Not whitelisted for EarthEngine access: trying anonymous access"